packages/dd-trace/src/appsec/waf/waf_manager.js

```diff
@@ -39,7 +39,8 @@ class WAFManager {
         this.ddwaf.createContext(),
         this.wafTimeout,
         this.ddwafVersion,
-        this.rulesVersion
+        this.rulesVersion,
+        this.ddwaf.knownAddresses
```

Comment:
no test for this ? should we make one ?

Reply:
I added a simple test for this property.